code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -266,3 +266,18 @@
 			else
 				to_chat(user, "<span class='warning'>You can't improve [D] any further!</span>")
 				return
+			if(damage_absorption["brute"] > 0.3)
```

Comment:
uh, what are you trying to accomplish here and why did you copy-paste code in its entirety from above?

Reply:
The code was a mess. It could be done in a single line. My bad
		`	if(D.icon_state != "ripley-open" && D.icon_state != "firefighter-open")`